src/ducks/settings/TogglePane.styl

```diff
@@ -0,0 +1,5 @@
+.TogglePane__main
+    margin-bottom 3rem
```

Comment:
Why do we need another class for this ? Can we use the spacing styles from cozy-ui ?